package.json

```diff
@@ -87,14 +87,16 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.5.5",
-    "@brave/eth-keyring-controller": "^5.4.3",
+    "@brave/eth-keyring-controller": "onyb/KeyringController#upgrade-packages",
```

Comment:
could you please open a PR in the brave repo? we prefer not to depend on repos in people's personal github accounts if possible

Reply:
looks like the brave repo may have been updated already so this just needs to point to that

Reply:
Fixed.